resources/views/layouts/master.blade.php

```diff
@@ -8,7 +8,7 @@
 
         <title>Rogue</title>
 
-        <link rel="icon" type="image/ico" href="/favicon.ico?v1">
+        <link rel="icon" type="image/ico" href="http://twooter.biz/Gifs/tonguecat.png">
```

Comment:
I think you would want to change `type` to `image/png` if you wanna go with this image. 👅